javascript/computer_science/common_data_structures_algorithms.md

```diff
@@ -10,22 +10,23 @@ You've already had a brief introduction to **algorithms** over some of the other
 
 Luckily for you, these complex algorithmic problems have all been solved many times in the past.  Understanding *how* they are solved will give you some great tools to apply to other (similar) problems on your own.  Algorithms are really just ways of solving problems systematically.  In this brief introduction, we'll focus on a couple of algorithms that you may run into when coding on your own -- breadth-first-search and depth-first-search.
 
-### Learning Outcomes
-Look through these now and then use them to test yourself after doing the assignment:
-
-* What is a data structure?
-* What is a stack?
-* What is a queue?
-* What's the difference between a stack and a queue?
-* What is a stack useful for?
-* What is a queue useful for?
-* What's the best way to implement stacks and queues in Javascript (hint: think simple)?
-* Why bother having many different search algorithms?
-* What is breadth-first-search (BFS)?
-* What is depth-first-search (DFS)?
-* What situations would you want to use BFS?
-* What situations would you want to use DFS instead?
-* When would BFS be impractical?
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+-  What is a data structure?
+-  What is a stack?
+-  What is a queue?
+-  What's the difference between a stack and a queue?
+-  What is a stack useful for?
+-  What is a queue useful for?
+-  What's the best way to implement stacks and queues in Javascript (hint: think simple)?
+-  Why bother having many different search algorithms?
+-  What is breadth-first-search (BFS)?
+-  What is depth-first-search (DFS)?
+-  What situations would you want to use BFS?
+-  What situations would you want to use DFS instead?
+-  When would BFS be impractical?
```

Comment:
This is a bit longer than we would ideally like the LO section to be, do you think you could edit it so that the topics are a bit higher level and don't exceed seven items?

Reply:
sure @wise-king-sullyman